ci/test_integration_espnet2.sh

```diff
@@ -11,48 +11,58 @@ cwd=$(pwd)
-    --asr-args "--model_conf extract_feats_in_collect_stats=false --max_epoch=1"
 
-echo "==== use_streaming, feats_type=raw, token_types=bpe, model_conf.extract_feats_in_collect_stats=False, normalize=utt_mvn ==="
-./run.sh --use_streaming true --ngpu 0 --stage 6 --stop-stage 13 --skip-upload false --feats-type "raw" --token-type "bpe" \
-    --feats_normalize "utterance_mvn" --lm-args "--max_epoch=1" --python "${python}" \
-    --asr-args "--model_conf extract_feats_in_collect_stats=false --max_epoch=1 --encoder=contextual_block_transformer --decoder=transformer
-                --encoder_conf block_size=40 --encoder_conf hop_size=16 --encoder_conf look_ahead=16"
+echo "=== feats_type=raw, token_types=char, compute_rtf=True, normalize=utt_mvn, use_lm=False ==="
+./run.sh --ngpu 0 --stage 10 --stop-stage 13 --skip-upload true --feats-type "raw" --token-type "char" \
+	 --feats_normalize "utterance_mvn" --use_lm false --python "${python}" ${asr_add_args} --compute_rtf true \
+	 --asr_tag "normalized" --asr-config ${train_conf} --inference-config ${decode_conf}
+
+echo "=== use_streaming, feats_type=raw, token_types=char, model_conf.extract_feats_in_collect_stats=False, normalize=utt_mvn ==="
+train_stream_conf=conf/train_streaming.yaml
+
+./run.sh --use_streaming true --ngpu 0 --stage 10 --stop-stage 13 --skip-upload true --feats-type "raw" --token-type "char" \
+	 --asr_tag "streaming" --feats_normalize "utterance_mvn" --python "${python}" \
+	 --asr-config ${train_stream_conf} --inference-config ${decode_conf} --lm-config ${lm_conf} ${asr_add_args}
 
 if python3 -c "import k2" &> /dev/null; then
-    echo "==== use_k2, num_paths > nll_batch_size, feats_type=raw, token_types=bpe, model_conf.extract_feats_in_collect_stats=False, normalize=utt_mvn ==="
-    ./run.sh --num_paths 500 --nll_batch_size 20 --use_k2 true --ngpu 0 --stage 12 --stop-stage 13 --skip-upload false --feats-type "raw" --token-type "bpe" \
-        --feats_normalize "utterance_mvn" --lm-args "--max_epoch=1" --python "${python}" \
-        --asr-args "--model_conf extract_feats_in_collect_stats=false --max_epoch=1"
-
-    echo "==== use_k2, num_paths == nll_batch_size, feats_type=raw, token_types=bpe, model_conf.extract_feats_in_collect_stats=False, normalize=utt_mvn ==="
-    ./run.sh --num_paths 20 --nll_batch_size 20 --use_k2 true --ngpu 0 --stage 12 --stop-stage 13 --skip-upload false --feats-type "raw" --token-type "bpe" \
-       --feats_normalize "utterance_mvn" --lm-args "--max_epoch=1" --python "${python}" \
-       --asr-args "--model_conf extract_feats_in_collect_stats=false --max_epoch=1"
-fi
+    decode_k2_conf=conf/decode_k2.yaml
+    asr_add_args+=" --use_k2 true --num_paths 5 --nll_batch_size 2 --use_nbest_rescoring false"
 
+    echo "=== use_k2, num_paths > nll_batch_size, feats_type=raw, token_types=char, model_conf.extract_feats_in_collect_stats=False, normalize=utt_mvn ==="
```

Comment:
Btw, do we need to test `num_paths > nll_batch_size` and `num_paths == nll_batch_size` here?

Also, sorry, I forgot k2 tests were not executed on my current machine (due to no compatible build available for my setup). I will fix it!